.github/workflows/distro-release.yml

```diff
@@ -45,3 +45,76 @@ jobs:
             "$MATRIX_FILE" > tmp.json
 
           echo "matrix=$(jq -c . tmp.json)" >> $GITHUB_OUTPUT
+
+  windows-build-test:
+    needs: [ config ]
+    uses: ./.github/workflows/build-test-windows.yml
+    with:
+      config_matrix:         ${{ needs.config.outputs.windows_x64_config_matrix }}
+      full_config_build:     ${{ needs.config.outputs.full_config_build == 'true' }}
+      internal_build:        ${{ needs.config.outputs.internal_build == 'true' }}
+      upload_test_artifacts: ${{ needs.config.outputs.upload_test_artifacts == 'true' }}
+      nuget_build:           ${{ needs.config.outputs.nuget_build == 'true' }}
```

Comment:
one other thing - nuget distribution required `vs22/release/msbuild` + `ubuntu20/x64/release`, will this requirement be fulfilled for releases?